_posts/2024-02-17-haveno-council-vote.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+permalink: /blog/:title/
+title:  "Haveno Council Vote"
+excerpt: Register to vote for the Haveno Council seats.
+author: Haveno Core Team
+---
+
+Hello all, this is information on registering to vote for Haveno Council's 3rd (possible 4th and 5th) seats.
+
+Currently requirements to register to vote is a working email and a well known online screen name from one of the given services in the form.
```

Comment:
Current requirements to register to vote are a working email and a known online screen name from one of the given services in the form.